documentation/4_Perspectives_of_the_Reference_Architecture_Model/4_3_Governance_Perspective/4_3_X_Goverance_for_Vocabularies.md

```diff
@@ -0,0 +1,67 @@
+or stability of concepts over a certain period. This involves the use of metrics for the maturity and selection
+of a suitable versioning scheme.
+
+Setting up governance of semantic standards is essential to ensure they are technically sound and reliable,
+but also to keep them in line with the changing needs of its users. Governance means setting guidelines for
+creating and maintaining the standards, as well as processes for resolving conflicts or inconsistencies that may arise. 
+
+The governance perspective comprises the following aspects:
+* Lifecycle of vocabularies: Vocabularies can pass through phases from exploration to retirement, which needs to be followed in order to aid users in selecting suitable vocabularies for their solutions.
+* Maturity of vocabularies: Given the development as a continuous process, a vocabulary can pass through different stages of maturity, including such as completeness of documentation or stability of concepts over a certain period. This involves metrics for the maturity and selection of a suitable versioning scheme.
+* Approval of vocabularies: Release of a vocabulary should be controlled by either a decision board and supported by automated model checking where possible.
+
+#### Open vs. centralized governance
+Governance can be set up in varying gradations of openness and centralization. The former promotes a more democratic and participatory process, where decision about the standards are made by a broad range of stakeholders. Governance in this sense is about ensuring that the standards are inclusive and reflective of the needs of all stakeholders.
+
+In centralized governance, the authority and responsibility often rest with a single organization, such as a government agency or an industry association. This organization has the power to create and maintain the standards, and often also to enforce them through regulations or other means. 
+
+A governance model leaning towards centralization can be more efficient, as it allows for quicker decision-making and can be more effective in ensuring compliance. However, because less effort is spent on fostering inclusiveness and participation, it runs a greater risk of overrepresenting some stakeholders while underrepresting the interests of others in the decision about the standards.
+
+#### Governance goals
+Whatever the approach, the goals of governance remain the same and can be summarized as: 
+- to ensure reliability of the vocabulary
+- to ensure quality of the vocabulary
+- to boost adoption of the vocabulary
+
+Decisions about the semantic standard are made through the governance organization, e.g., decision to work start development work, approval of changes in the new version, and release of said version through publication on the web. In open standardization, the work is carried out by its members as much as possible, while the goverannce organization acts as coordinator and oversees progress.
```

Comment:
... is carried out by its members  ... 
who or what is 'its' referring to exactly?